README.md

```diff
@@ -292,31 +292,33 @@ If you have received the package and only want to build and run the docker conta
     ```
 
 ### Running the Signing Service
+Set `ETHEREUM_RPIVATE_KEY`, `SIGNING_SERVICE_PRIVATE_KEY` and if you want `SENTRY_DSN` to bash variables encoded in base64 before proceeding.
```

Comment:
Please reword and put it in the section after the command:
```
This assumes that:
(...)
- There's a shell variable called `ETHEREUM_RPIVATE_KEY` and it contains base64-encoded private key of the Concent contract.
- There's a shell variable called `SIGNING_SERVICE_PRIVATE_KEY` and it contains base64-encoded private key for signing Golem Messages created by the Signing Service.
- There's a shell variable called `SENTRY_DSN` that contains the secret ID that allows submitting crash report to a [Sentry](https://sentry.io) project.
(...)
```

Reply:
Done